README.md

```diff
@@ -119,9 +128,14 @@ python main.py
 
 #### Using Docker Compose
 
-2. Run the following command to build and run the Docker image:
-```bash
-docker-compose up
+Run the following command to build and run the Docker image:
+```shell
+docker compose up
+```
+
+**Note:** After updating `.env` file, you have to rebuild the image:
+```shell
+docker compose up --build
```

Comment:
This is not necessary, since the file is mounted when the container is started.

More correctly:
After the change `.env` file restart docker container:
```shell
docker-compose down
docker-compose up
```
If they have been changed `.py` files, then you need to rebuild the docker image:
```shell
docker-compose down
docker compose up --build
```

Reply:
aaw, I see. I should have checked the compose file more accurately.
Just pushed a fix.

Please note that I kept the new docker compose command syntax standard which came from the built-in compose feature.